stdlib/public/Concurrency/TaskGroup.cpp

```diff
@@ -514,9 +507,6 @@ void TaskGroupImpl::destroy() {
     taskDequeued = readyQueue.dequeue(item);
   }
   mutex.unlock(); // TODO: remove fragment lock, and use status for synchronization
-
-  // TODO: get the parent task, do we need to store it?
-  swift_task_dealloc(this);
```

Comment:
oh fantastic, so we don't have to do this dance anymore since it's managed by the Lifetime start/end... that's great 👍